src/classes/Commands.ts

```diff
@@ -1227,6 +1230,100 @@ export = class Commands {
         });
     }
 
+    private deleteCommand(steamID: SteamID, message: string): void {
+        const params = CommandParser.parseParams(CommandParser.removeCommand(message));
+
+        if (params.name !== undefined || params.item !== undefined) {
+            this.bot.sendMessage(
+                steamID,
+                `Please only use sku property.
+
+                Below are some common items to delete:
+                • Smissamas Sweater: 16391;15;untradable;w1;pk391
+                • Soul Gargoyle: 5826;6;uncraftable;untradable
+                • Noice Maker - TF Birthday: 536;6;untradable
+                • Bronze Dueling Badge: 242;6;untradable
+                • Silver Dueling Badge: 243;6;untradable
+                • Gold Dueling Badge: 244;6;untradable
+                • Platinum Dueling Badge: 245;6;untradable
+                • Mercenary: 166;6;untradable
+                • Soldier of Fortune: 165;6;untradable
+                • Grizzled Veteran: 164;6;untradable
+                • Primeval Warrior: 170;6;untradable
+                • Professor Speks: 343;6;untradable
+                • Mann Co. Cap: 261;6;untradable
+                • Mann Co. Online Cap: 994;6;untradable
+                • Proof of Purchase: 471;6;untradable
+                • Mildly Disturbing Halloween Mask: 115;6;untradable
+                • Seal Mask: 582;6;untradable
+                • Pyrovision Goggles: 743;6;untradable
+                • Giftapult: 5083;6;untradable
+                • Spirit Of Giving: 655;11;untradable
+                • Party Hat: 537;6;untradable
+                • Name Tag: 5020;6;untradable
+                • Description Tag: 5044;6;untradable
+                • Ghastly Gibus: 584;6;untradable
+                • Ghastlier Gibus: 279;6;untradable
+                • Power Up Canteen: 489;6;untradable
+                • Bombinomicon: 583;6;untradable
+                • Skull Island Topper: 941;6;untradable
+                • Spellbook Page: 8935;6;untradable
+                • Gun Mettle Campaign Coin: 5809;6;untradable
+                • MONOCULUS!: 581;6;untradable
+                
+                Or other items, please refer here: https://bit.ly/3gZQxFQ (defindex)`
+            );
+            return;
+        }
+
+        if (params.sku === undefined) {
+            this.bot.sendMessage(steamID, 'Missing item sku');
+            return;
+        }
+
+        let uncraft = false;
+        if (params.sku.includes('uncraftable')) {
+            params.sku = params.sku.replace(';uncraftable', '');
+            uncraft = true;
+        }
+
+        let untrade = false;
+        if (params.sku.includes('untradable')) {
+            params.sku = params.sku.replace(';untradable', '');
+            untrade = true;
+        }
+
+        const item = SKU.fromString(params.sku);
+
+        if (uncraft) {
```

Comment:
It might be helpful to make the property always be present.
Currently, these are tri-state properties.
`true|false|[missing]`
Since this property is new, I'd recommend making it bi-state from the start.
```TS
item.craftable = !uncraft;
```